hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneShellHA.java

```diff
@@ -963,20 +963,4 @@ public void testListVolumeBucketKeyShouldPrintValidJsonArray()
     testVolumes.forEach(vol -> execute(ozoneShell, new String[] {
         "volume", "delete", vol}));
   }
-
-  @Test
-  public void testClientBucketLayoutValidation() throws Exception {
-    String[] args = new String[]{
-        "bucket", "create", "o3://" + omServiceId + "/volume7" + "/bucketTest1",
-        "--layout", "LEGACY"
-    };
-    try {
-      execute(ozoneShell, args);
-      Assert.fail("Should throw exception on unsupported bucket layouts!");
-    } catch (Exception e) {
-      GenericTestUtils.assertExceptionContains(
-          "expected one of [FILE_SYSTEM_OPTIMIZED, OBJECT_STORE] ",
-          e);
-    }
-  }
```

Comment:
This test is no longer valid since `LEGACY` is now added to the list of valid bucket layouts.

Reply:
@JyotinderSingh I'd like to not to expose LEGACY layout and internally defaulting to LEGACY if not given/passed.

Reply:
LEGACY is no longer listed as the default value for the default bucket layout config key